--- percona-postgresql-11.spec	2019-08-08 11:37:21.777945554 +0000
+++ percona-postgresql-11.spec	2019-08-08 11:36:37.341201908 +0000
@@ -274,7 +274,7 @@
 
 BuildRoot:	%{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 Provides:	%{sname}
-
+Conflicts:	%{sname}
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
@@ -297,6 +297,7 @@
 Summary:	The shared libraries required for any PostgreSQL clients
 Group:		Applications/Databases
 Provides:	postgresql-libs = %{pgmajorversion}
+Conflicts:	postgresql-libs = %{pgmajorversion}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -334,6 +335,7 @@
 Requires:	/usr/sbin/useradd, /sbin/chkconfig
 %endif
 Provides:	postgresql-server
+Conflicts:	postgresql-server
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -350,6 +352,7 @@
 Summary:	Extra documentation for PostgreSQL
 Group:		Applications/Databases
 Provides:	postgresql-docs
+Conflicts:	postgresql-docs
 
 %description docs
 The postgresql%{pgmajorversion}-docs package includes the SGML source for the documentation
@@ -364,6 +367,7 @@
 Requires:	%{name}%{?_isa} = %{version}-%{release}
 Requires:	%{name}-libs%{?_isa} = %{version}-%{release}
 Provides:	postgresql-contrib
+Conflicts:	postgresql-contrib
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -399,6 +403,7 @@
 %endif
 
 Provides:	postgresql-devel
+Conflicts:	postgresql-devel
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -427,6 +432,7 @@
 %endif
 %endif
 Provides:	postgresql-llvmjit
+Conflicts:	postgresql-llvmjit
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -451,6 +457,7 @@
 %endif
 Obsoletes:	postgresql%{pgmajorversion}-pl
 Provides:	postgresql-plperl
+Conflicts:	postgresql-plperl
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -472,7 +479,9 @@
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
 Obsoletes:	%{name}-pl
 Provides:	postgresql-plpython
+Conflicts:	postgresql-plpython
 Provides:	%{name}-plpython2%{?_isa} = %{version}-%{release}
+Conflicts:	%{name}-plpython2%{?_isa} = %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -494,6 +503,7 @@
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
 Obsoletes:	%{name}-pl
 Provides:	postgresql-plpython3
+Conflicts:	postgresql-plpython3
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -516,6 +526,7 @@
 Requires:	tcl
 Obsoletes:	%{name}-pl
 Provides:	postgresql-pltcl
+Conflicts:	postgresql-pltcl
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -535,6 +546,7 @@
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
 Requires:	%{name}-devel%{?_isa} = %{version}-%{release}
 Provides:	postgresql-test
+Conflicts:	postgresql-test
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
