--- control	2019-08-12 17:25:40.632540037 +0300
+++ control	2019-08-13 12:25:59.507649882 +0300
@@ -1,28 +1,20 @@
-Source: postgresql-common
+Source: percona-postgresql-common
 Section: database
 Priority: optional
-Maintainer: Debian PostgreSQL Maintainers <team+postgresql@tracker.debian.org>
-Uploaders:
- Martin Pitt <mpitt@debian.org>,
- Christoph Berg <myon@debian.org>,
- Peter Eisentraut <petere@debian.org>,
-Standards-Version: 4.3.0
-Vcs-Git: https://salsa.debian.org/postgresql/postgresql-common.git
-Vcs-Browser: https://salsa.debian.org/postgresql/postgresql-common
-Build-Depends:
+Maintainer: Percona Development Team <info@percona.com>
  debhelper (>= 9),
  debhelper (>= 10.1) | dh-systemd (>= 1.19),
  libreadline-dev,
  lsb-release,
 
-Package: postgresql-common
+Package: percona-postgresql-common
 Architecture: all
 Multi-Arch: foreign
 Depends:
  adduser,
  debconf (>= 0.5.00) | debconf-2.0,
  lsb-base (>= 3.0-3),
- postgresql-client-common (= ${source:Version}),
+ percona-postgresql-client-common (= ${source:Version}),
  procps,
  ssl-cert (>= 1.0.11),
  ucf,
@@ -34,6 +26,7 @@
  libjson-perl,
 Breaks:
  postgresql-9.1 (<< 9.1.1-3~),
+ postgresql-common,
  systemd (<< 204),
 # pg_buildext moved pg-s-d-all in 204
  postgresql-server-dev-all (<< 204),
@@ -55,7 +48,7 @@
  outer joins, multiversion concurrency control, and user-defined types
  and functions.
 
-Package: postgresql-client-common
+Package: percona-postgresql-client-common
 Architecture: all
 Multi-Arch: foreign
 Depends:
@@ -65,6 +58,7 @@
 Recommends:
  lsb-release,
  ${readline:Recommends},
+Breaks: postgresql-client-common
 Description: manager for multiple PostgreSQL client versions
  The postgresql-client-common package provides a structure under which
  multiple versions of PostgreSQL client programs may be installed at
@@ -80,17 +74,18 @@
  outer joins, multiversion concurrency control, and user-defined types
  and functions.
 
-Package: postgresql-server-dev-all
+Package: percona-postgresql-server-dev-all
 Architecture: all
 Depends:
  dctrl-tools,
  lsb-release,
  make,
- postgresql-common (>= 117~),
+ percona-postgresql-common (>= 117~),
  ${misc:Depends},
  ${server-dev-all-depends},
+Breaks: postgresql-server-dev-all
 Description: extension build tool for multiple PostgreSQL versions
- The postgresql-server-dev-all package provides the pg_buildext script for
+ The percona-postgresql-server-dev-all package provides the pg_buildext script for
  simplifying packaging of a PostgreSQL extension supporting multiple major
  versions of the product.
  .
@@ -101,13 +96,14 @@
  outer joins, multiversion concurrency control, and user-defined types
  and functions.
 
-Package: postgresql
+Package: percona-postgresql
 Architecture: all
 Depends:
- postgresql-${default-version},
+ percona-postgresql-${default-version},
  ${misc:Depends},
 Suggests:
- postgresql-doc,
+ percona-postgresql-doc,
+Breaks: postgresql
 Description: object-relational SQL database (supported version)
  This metapackage always depends on the currently supported PostgreSQL
  database server version.
@@ -120,12 +116,13 @@
  concurrency control.  Graphical user interfaces and bindings for many
  programming languages are available as well.
 
-Package: postgresql-client
+Package: percona-postgresql-client
 Architecture: all
 Multi-Arch: foreign
 Depends:
- postgresql-client-${default-version},
+ percona-postgresql-client-${default-version},
  ${misc:Depends},
+Breaks: postgresql-client
 Description: front-end programs for PostgreSQL (supported version)
  This metapackage always depends on the currently supported database
  client package for PostgreSQL.
@@ -138,13 +135,14 @@
  concurrency control.  Graphical user interfaces and bindings for many
  programming languages are available as well.
 
-Package: postgresql-doc
+Package: percona-postgresql-doc
 Architecture: all
 Multi-Arch: foreign
 Section: doc
 Depends:
- postgresql-doc-${default-version},
+ percona-postgresql-doc-${default-version},
  ${misc:Depends},
+Breaks: postgresql-doc
 Description: documentation for the PostgreSQL database management system
  This metapackage always depends on the currently supported PostgreSQL
  database documentation package.
@@ -157,11 +155,12 @@
  concurrency control.  Graphical user interfaces and bindings for many
  programming languages are available as well.
 
-Package: postgresql-contrib
+Package: percona-postgresql-contrib
 Architecture: all
 Depends:
- postgresql-contrib-${default-version},
+ percona-postgresql-contrib-${default-version},
  ${misc:Depends},
+Breaks: postgresql-contrib
 Description: additional facilities for PostgreSQL (supported version)
  This metapackage always depends on the currently supported PostgreSQL
  database contrib package.
@@ -174,14 +173,15 @@
  concurrency control.  Graphical user interfaces and bindings for many
  programming languages are available as well.
 
-Package: postgresql-all
+Package: percona-postgresql-all
 Architecture: all
 Depends:
  libecpg-dev,
  libpq-dev,
- postgresql-server-dev-all,
+ percona-postgresql-server-dev-all,
  ${postgresql-all-depends},
  ${misc:Depends},
+Breaks: postgresql-server-dev-all
 Description: metapackage depending on all PostgreSQL server packages
  This metapackage depends on all PostgreSQL server packages, in all
  supported versions (excluding documentation and debug symbols). It
