--- control    2019-10-01 17:58:05.335136015 -0400
+++ control    2019-10-01 17:55:36.299136015 -0400
@@ -1,12 +1,7 @@
-Source: postgresql-11
+Source: percona-postgresql-11
 Section: database
 Priority: optional
-Maintainer: Debian PostgreSQL Maintainers <team+postgresql@tracker.debian.org>
-Uploaders:
- Martin Pitt <mpitt@debian.org>,
- Peter Eisentraut <petere@debian.org>,
- Christoph Berg <myon@debian.org>
-Standards-Version: 4.3.0
+Maintainer: Percona Development Team <info@percona.com>
 Build-Depends:
  bison,
  clang-9 [!alpha !hppa !hurd-i386 !ia64 !kfreebsd-amd64 !kfreebsd-i386 !m68k !powerpc !riscv64 !sh4 !sparc64 !x32],
@@ -41,8 +36,6 @@
  xsltproc,
  zlib1g-dev | libz-dev
 Homepage: http://www.postgresql.org/
-Vcs-Browser: https://salsa.debian.org/postgresql/postgresql
-Vcs-Git: https://salsa.debian.org/postgresql/postgresql.git -b 11
 
 Package: libpq-dev
 Architecture: any
@@ -51,7 +44,7 @@
  libpq5 (= ${binary:Version}),
  ${misc:Depends},
  ${shlibs:Depends}
-Suggests: postgresql-doc-11
+Suggests: percona-postgresql-doc-11
 Description: header files for libpq5 (PostgreSQL library)
  Header files and static library for compiling C programs to link
  with the libpq library in order to communicate with a PostgreSQL
@@ -132,17 +125,19 @@
  .
  PostgreSQL is an object-relational SQL database management system.
 
-Package: postgresql-11
+Package: percona-postgresql-11
 Architecture: any
 Depends:
  locales | locales-all,
- postgresql-client-11,
- postgresql-common (>= 194~),
+ percona-postgresql-client-11,
+ percona-postgresql-common (>= 194~),
  ssl-cert,
  tzdata,
  ${misc:Depends},
  ${shlibs:Depends}
-Provides: postgresql-contrib-11
+Provides: percona-postgresql-contrib-11 (= 2:11.6-2~), postgresql-contrib-11(= 2:11.6-2~), postgresql-11(= 2:11.6-2~), postgresql (= 2:11.6-2~)
+Replaces: postgresql-contrib-11 (<= 11.6-2~), postgresql-contrib-11 (<= 11.6-2~), postgresql-11 (<= 11.6-2~), postgresql (<= 11.6-2~)
+Conflicts: postgresql-contrib-11 (<= 11.6-2~), postgresql-contrib-11 (<= 11.6-2~), postgresql-11 (<= 11.6-2~), postgresql (<= 11.6-2~)
 Recommends: sysstat
 Breaks:
  postgresql-11-citus (<< 8.0.0.PGDG-2~),
@@ -152,6 +147,7 @@
  postgresql-11-plsh (<< 1.20171014-3~),
  postgresql-11-rum (<< 1.3.2-4~),
  postgresql-11-wal2json (<< 1.0-5~),
+ postgresql-contrib-11 (<= 11.6-2~), postgresql-contrib-11 (<= 11.6-2~), postgresql-11 (<= 11.6-2~), postgresql (<= 11.6-2~)
 Description: object-relational SQL database, version 11 server
  PostgreSQL is a powerful, open source object-relational database
  system. It is fully ACID compliant, has full support for foreign
@@ -176,19 +172,20 @@
 #dbg# .
 #dbg# PostgreSQL is an object-relational SQL database management system.
 
-Package: postgresql-client-11
+Package: percona-postgresql-client-11
 Architecture: any
 Multi-Arch: foreign
 Depends:
  libpq5 (>= ${source:Upstream-Version}),
- postgresql-client-common (>= 182~),
+ percona-postgresql-client-common (>= 182~),
  sensible-utils,
  ${misc:Depends},
  ${shlibs:Depends}
-Suggests: postgresql-11, postgresql-doc-11
-Provides: postgresql-client
+Suggests: percona-postgresql-11, percona-postgresql-doc-11
+Provides: percona-postgresql-client, postgresql-client (= 2:11.6-2~), postgresql-client-11 (= 2:11.6-2~)
 Conflicts: postgresql-server-dev-11 (<< 11.6-2~)
-Replaces: postgresql-server-dev-11 (<< 11.6-2~)
+Breaks: postgresql-server-dev-11 (<< 11.6-2~)
+Replaces: postgresql-server-dev-11 (<= 11.6-2~), postgresql-client (<= 11.6-2~), postgresql-client-11 (<= 11.6-2~)
 Description: front-end programs for PostgreSQL 11
  This package contains client and administrative programs for
  PostgreSQL: these are the interactive terminal client psql and
@@ -202,19 +199,22 @@
  .
  PostgreSQL is an object-relational SQL database management system.
 
-Package: postgresql-server-dev-11
+Package: percona-postgresql-server-dev-11
 Architecture: any
 Section: libdevel
 Depends:
  clang-9 [!alpha !hppa !hurd-i386 !ia64 !kfreebsd-amd64 !kfreebsd-i386 !m68k !powerpc !riscv64 !sh4 !sparc64 !x32],
  libpq-dev (>= 11~~),
  llvm-9-dev [!alpha !hppa !hurd-i386 !ia64 !kfreebsd-amd64 !kfreebsd-i386 !m68k !powerpc !riscv64 !sh4 !sparc64 !x32],
- postgresql-client-11,
- postgresql-common (>= 142~),
+ percona-postgresql-client-11, percona-postgresql-common (>= 142~),
  ${misc:Depends},
  ${shlibs:Depends}
-Breaks:
- postgresql-server-dev-all (<< 181~)
+#Breaks:
+# postgresql-server-dev-all (<< 181~)
+Provides: postgresql-server-dev-all (= 2:11.6-2~), postgresql-server-dev-all-11 (= 2:11.6-2~)
+Replaces: postgresql-server-dev-all (<= 11.6-2~), postgresql-server-dev-all-11 (<= 11.6-2~)
+Conflicts: postgresql-server-dev-all (<= 11.6-2~), postgresql-server-dev-all-11 (<= 11.6-2~)
+Breaks: postgresql-server-dev-all (<= 11.6-2~), postgresql-server-dev-all-11 (<= 11.6-2~), postgresql-server-dev-all (<< 181~)
 Description: development files for PostgreSQL 11 server-side programming
  Header files for compiling SSI code to link into PostgreSQL's backend; for
  example, for C functions to be called from SQL.
@@ -225,7 +225,7 @@
  .
  PostgreSQL is an object-relational SQL database management system.
 
-Package: postgresql-doc-11
+Package: percona-postgresql-doc-11
 Architecture: all
 Multi-Arch: foreign
 Section: doc
@@ -237,14 +237,17 @@
  .
  PostgreSQL is an object-relational SQL database management system.
 
-Package: postgresql-plperl-11
+Package: percona-postgresql-plperl-11
 Architecture: any
 Depends:
  perl,
- postgresql-11 (= ${binary:Version}),
+ percona-postgresql-11 (= ${binary:Version}),
  ${misc:Depends},
  ${shlibs:Depends}
-Provides: postgresql-plperl
+Provides: percona-postgresql-plperl, postgresql-plperl (= 2:11.6-2~), postgresql-plperl-11 (= 2:11.6-2~)
+Replaces: postgresql-plperl (<= 11.6-2~), postgresql-plperl-11 (<= 11.6-2~)
+Conflicts: postgresql-plperl (<= 11.6-2~), postgresql-plperl-11 (<= 11.6-2~)
+Breaks: postgresql-plperl (<= 11.6-2~), postgresql-plperl-11 (<= 11.6-2~)
 Description: PL/Perl procedural language for PostgreSQL 11
  PL/Perl enables an SQL developer to write procedural language functions
  for PostgreSQL 11 in Perl.  You need this package if you have any
@@ -269,13 +272,15 @@
 #py2# .
 #py2# PostgreSQL is an object-relational SQL database management system.
 
-Package: postgresql-plpython3-11
+Package: percona-postgresql-plpython3-11
 Architecture: any
 Depends:
- postgresql-11 (= ${binary:Version}),
+ percona-postgresql-11 (= ${binary:Version}),
  ${misc:Depends},
  ${shlibs:Depends}
-Provides: postgresql-plpython3
+Provides: percona-postgresql-plpython3, postgresql-plpython3 (= 2:11.6-2~), postgresql-11-plpython3 (= 2:11.6-2~)
+Replaces: postgresql-plpython3 (<= 11.6-2~), postgresql-11-plpython3 (<= 11.6-2~)
+Conflicts: postgresql-plpython3 (<= 11.6-2~), postgresql-11-plpython3 (<= 11.6-2~)
 Description: PL/Python 3 procedural language for PostgreSQL 11
  PL/Python 3 enables an SQL developer to write procedural language functions
  for PostgreSQL 11 in Python 3.  You need this package if you have any
@@ -283,13 +288,16 @@
  .
  PostgreSQL is an object-relational SQL database management system.
 
-Package: postgresql-pltcl-11
+Package: percona-postgresql-pltcl-11
 Architecture: any
 Depends:
- postgresql-11 (= ${binary:Version}),
+ percona-postgresql-11 (= ${binary:Version}),
  ${misc:Depends},
  ${shlibs:Depends}
-Provides: postgresql-pltcl
+Provides: percona-postgresql-pltcl, postgresql-pltcl (= 2:11.6-2~), postgresql-11-pltcl (= 2:11.6-2~)
+Replaces: postgresql-pltcl (<= 11.6-2~), postgresql-11-pltcl (<= 11.6-2~)
+Conflicts: postgresql-pltcl (<= 11.6-2~), postgresql-11-pltcl (<= 11.6-2~)
+Breaks: postgresql-pltcl (<= 11.6-2~), postgresql-11-pltcl (<= 11.6-2~)
 Description: PL/Tcl procedural language for PostgreSQL 11
  PL/Tcl enables an SQL developer to write procedural language functions
  for PostgreSQL 11 in Tcl.  You need this package if you have any
