--- maintscripts-functions	2019-07-04 12:13:47.057670000 +0000
+++ maintscripts-functions	2019-07-04 12:14:12.249670000 +0000
@@ -125,7 +125,7 @@
 }
 
 configure_client_version() {
-    _link_manpages "$1" psql.1.gz "postgresql-client-$1"
+    _link_manpages "$1" psql.1.gz "percona-postgresql-client-$1"
 }
 
 remove_client_version() {
@@ -133,16 +133,16 @@
 }
 
 configure_contrib_version() {
-    _link_manpages "$1" postmaster.1.gz "postgresql-$1" "postgresql-contrib-$1"
+    _link_manpages "$1" postmaster.1.gz "ercona-postgresql-$1" "percona-postgresql-contrib-$1"
 }
 
 remove_contrib_version() {
     # we still want to retain the alternatives for the server packages
-    _link_manpages "$1" postmaster.1.gz "postgresql-$1"
+    _link_manpages "$1" postmaster.1.gz "percona-postgresql-$1"
 }
 
 configure_doc_version() {
-    _link_manpages "$1" SPI_connect.3.gz "postgresql-doc-$1"
+    _link_manpages "$1" SPI_connect.3.gz "percona-postgresql-doc-$1"
 }
 
 remove_doc_version() {
@@ -232,13 +232,13 @@
 	  pg_dropcluster ${MAJOR_VER} ${cluster}.old
 	  rm -rf ${VARTMPDIR}
 	EOF
-        db_fset postgresql-common/catversion-bump seen false
-        db_subst postgresql-common/catversion-bump version $MAJOR_VER
-        db_subst postgresql-common/catversion-bump cluster $cluster
-        db_subst postgresql-common/catversion-bump db_catversion $DB_CATVERSION
-        db_subst postgresql-common/catversion-bump new_catversion $NEW_CATVERSION
-        db_subst postgresql-common/catversion-bump vartmpdir $VARTMPDIR
-        db_input high postgresql-common/catversion-bump || true
+        db_fset percona-postgresql-common/catversion-bump seen false
+        db_subst percona-postgresql-common/catversion-bump version $MAJOR_VER
+        db_subst percona-postgresql-common/catversion-bump cluster $cluster
+        db_subst percona-postgresql-common/catversion-bump db_catversion $DB_CATVERSION
+        db_subst percona-postgresql-common/catversion-bump new_catversion $NEW_CATVERSION
+        db_subst percona-postgresql-common/catversion-bump vartmpdir $VARTMPDIR
+        db_input high percona-postgresql-common/catversion-bump || true
         db_go || true
     done
 }
@@ -246,7 +246,7 @@
 # start debconf if we are in the server's postinst (can't run from a function)
 if [ "${DPKG_MAINTSCRIPT_NAME:-}" = "postinst" ] && [ "$1" = "configure" ]; then
     case $DPKG_MAINTSCRIPT_PACKAGE in
-        postgresql-[89].?|postgresql-[1-9]?)
+        percona-postgresql-[89].?|percona-postgresql-[1-9]?)
             . /usr/share/debconf/confmodule
             ;;
     esac
