--- percona-postgresql-11.spec	2019-12-13 13:06:53.448962404 +0200
+++ percona-postgresql-11.spec	2019-12-13 13:06:13.889127417 +0200
@@ -2,6 +2,7 @@
 %global packageversion 110
 %global prevmajorversion 10
 %global sname postgresql
+%global vname postgresql11
 %global	pgbaseinstdir	/usr/pgsql-%{pgmajorversion}
 
 %global beta 0
@@ -80,13 +81,13 @@
 %endif
 
 Summary:	PostgreSQL client programs and libraries
-Name:		%{sname}%{pgmajorversion}
+Name:		percona-postgresql%{pgmajorversion}
 Version:	11.6
-Release:	2PGDG%{?dist}
+Release:	2%{?dist}
 License:	PostgreSQL
 Url:		https://www.postgresql.org/
 
-Source0:	https://download.postgresql.org/pub/source/v%{version}/postgresql-%{version}.tar.bz2
+Source0:	percona-postgresql-%{version}.tar.gz
 Source4:	%{sname}-%{pgmajorversion}-Makefile.regress
 Source5:	%{sname}-%{pgmajorversion}-pg_config.h
 %if %{systemd_enabled}
@@ -300,6 +301,9 @@
 Requires:	openssl-libs >= 1.0.2k
 %endif
 %endif
+Provides:	%{vname}-libs = %{pgmajorversion} postrgesql-libs
+Conflicts:	postgresql-libs = %{pgmajorversion}
+Conflicts:	%{vname}-libs = %{pgmajorversion}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -336,6 +340,9 @@
 Requires:	/usr/sbin/useradd, /sbin/chkconfig
 %endif
 Provides:	postgresql-server >= %{version}-%{release}
+Provides:	%{vname}-server
+Conflicts:	postgresql-server
+Conflicts:	%{vname}-server
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -351,6 +358,9 @@
 %package docs
 Summary:	Extra documentation for PostgreSQL
 Provides:	postgresql-docs >= %{version}-%{release}
+Provides:	%{vname}-docs
+Conflicts:	postgresql-docs
+Conflicts:	%{vname}-docs
 
 %description docs
 The postgresql%{pgmajorversion}-docs package includes the SGML source for the documentation
@@ -364,6 +374,9 @@
 Requires:	%{name}%{?_isa} = %{version}-%{release}
 Requires:	%{name}-libs%{?_isa} = %{version}-%{release}
 Provides:	postgresql-contrib >= %{version}-%{release}
+Provides:	%{vname}-contrib
+Conflicts:	postgresql-contrib
+Conflicts:	%{vname}-contrib
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -398,6 +411,8 @@
 %endif
 
 Provides:	postgresql-devel >= %{version}-%{release}
+Provides:	%{vname}-devel
+Conflicts:	%{vname}-devel
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -425,6 +440,9 @@
 %endif
 %endif
 Provides:	postgresql-llvmjit >= %{version}-%{release}
+Provides:	%{vname}-llvmjit
+Conflicts:	postgresql-llvmjit
+Conflicts:	%{vname}-llvmjit
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -448,6 +466,9 @@
 %endif
 Obsoletes:	postgresql%{pgmajorversion}-pl <= %{version}-%{release}
 Provides:	postgresql-plperl >= %{version}-%{release}
+Provides:	%{vname}-plperl
+Conflicts:	postgresql-plperl
+Conflicts:	%{vname}-plperl
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -474,6 +495,9 @@
 %else
 Requires:	python2-libs
 %endif
+Conflicts:	postgresql-plpython
+Provides:	%{name}-plpython2%{?_isa} = %{version}-%{release}
+Conflicts:	%{name}-plpython2%{?_isa} = %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -495,6 +519,9 @@
 Obsoletes:	%{name}-pl <= %{version}-%{release}
 Provides:	postgresql-plpython3 >= %{version}-%{release}
 Requires:	python3-libs
+Provides:	%{vname}-plpython3
+Conflicts:	postgresql-plpython3
+Conflicts:	%{vname}-plpython3
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -516,6 +543,9 @@
 Requires:	tcl
 Obsoletes:	%{name}-pl <= %{version}-%{release}
 Provides:	postgresql-pltcl >= %{version}-%{release}
+Provides:	%{vname}-pltcl
+Conflicts:	postgresql-pltcl
+Conflicts:	%{vname}-pltcl
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -534,6 +564,9 @@
 Requires:	%{name}-server%{?_isa} = %{version}-%{release}
 Requires:	%{name}-devel%{?_isa} = %{version}-%{release}
 Provides:	postgresql-test >= %{version}-%{release}
+Provides:	%{vname}-test
+Conflicts:	postgresql-test
+Conflicts:	%{vname}-test
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -549,7 +582,7 @@
 %global __perl_requires %{SOURCE16}
 
 %prep
-%setup -q -n %{sname}-%{version}
+%setup -q -n percona-postgresql-%{version}
 %patch1 -p0
 %patch3 -p0
 %patch5 -p0
@@ -855,6 +888,7 @@
 	%{__make} all
 	popd
 %endif
+pushd doc/src; make all; popd
 
 %install
 %{__rm} -rf %{buildroot}
