--- control	2020-04-03 11:21:28.929467521 +0300
+++ control	2020-04-03 11:17:25.094701708 +0300
@@ -1,8 +1,7 @@
-Source: pgbackrest
+Source: percona-pgbackrest
 Section: database
 Priority: optional
-Maintainer: Debian PostgreSQL Maintainers <team+postgresql@tracker.debian.org>
-Uploaders: Adrian Vondendriesch <adrian.vondendriesch@credativ.de>
+Maintainer: Percona Development Team <info@percona.com>
 Rules-Requires-Root: no
 Build-Depends: debhelper (>= 9),
                libbz2-dev,
@@ -21,9 +20,10 @@
 Vcs-Git: https://salsa.debian.org/postgresql/pgbackrest.git
 Vcs-Browser: https://salsa.debian.org/postgresql/pgbackrest
 
-Package: pgbackrest
+Package: percona-pgbackrest
 Architecture: any
-Depends: postgresql-common, ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Depends: postgresql-common | percona-postgresql-common , ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Provides: pgbackrest
 Suggests: pgbackrest-doc
 Description: Reliable PostgreSQL Backup & Restore
  pgBackRest is a simple, reliable backup and restore system for PostgreSQL
@@ -31,11 +31,12 @@
  .
  This package contains all required binaries, scripts and libraries.
 
-Package: pgbackrest-doc
+Package: percona-pgbackrest-doc
 Architecture: all
 Multi-Arch: foreign
 Section: doc
 Depends: ${misc:Depends}
+Provides: pgbackrest-doc
 Description: Documentation package for pgbackrest
  pgBackRest is a simple, reliable backup and restore system for PostgreSQL
  that can seamlessly scale up to the largest databases and workloads.
