--- rules	2019-08-06 13:16:37.465457889 +0000
+++ rules	2019-08-06 13:15:50.469457889 +0000
@@ -21,7 +21,12 @@
 export DH_PIP_EXTRA_ARGS=-I
 
 %:
-	dh $@ --no-test --use-system-packages --buildsystem=dh_virtualenv
+	dh $@ --with python-virtualenv --python=python3
+
+override_dh_virtualenv:
+	dh_virtualenv --python=python3 --install-suffix=patroni --use-system-packages --no-test
+
+override_dh_strip:
 
 override_dh_auto_test:
 
