--- etc/pgbouncer.ini.old	2019-08-27 10:08:01.128201441 +0100
+++ etc/pgbouncer.ini	2019-08-27 10:08:18.820306554 +0100
@@ -127,10 +127,10 @@
 ;;;
 
 ;; comma-separated list of users who are allowed to change settings
-;admin_users = user2, someadmin, otheradmin
+admin_users = postgres
 
 ;; comma-separated list of users who are just allowed to use SHOW command
-;stats_users = stats, root
+stats_users = stats, postgres
 
 ;;;
 ;;; Pooler personality questions
