--- control.orig	2020-04-02 19:34:32.081450822 +0000
+++ control	2020-04-02 19:38:30.847553623 +0000
@@ -1,20 +1,13 @@
-Source: postgresql-common
+Source: percona-postgresql-common
 Section: database
 Priority: optional
-Maintainer: Debian PostgreSQL Maintainers <team+postgresql@tracker.debian.org>
-Uploaders:
- Martin Pitt <mpitt@debian.org>,
- Christoph Berg <myon@debian.org>,
- Peter Eisentraut <petere@debian.org>,
-Standards-Version: 4.4.0
-Vcs-Git: https://salsa.debian.org/postgresql/postgresql-common.git
-Vcs-Browser: https://salsa.debian.org/postgresql/postgresql-common
+Maintainer: Percona Development Team <info@percona.com>
 Build-Depends:
  debhelper (>= 9),
  debhelper (>= 10.1) | dh-systemd (>= 1.19),
  libreadline-dev,
 
-Package: postgresql-common
+Package: percona-postgresql-common
 Architecture: all
 Multi-Arch: foreign
 Depends:
@@ -22,7 +15,7 @@
  debconf (>= 0.5.00) | debconf-2.0,
  lsb-base (>= 3.0-3),
  perl (>= 5.14),
- postgresql-client-common (= ${source:Version}),
+ percona-postgresql-client-common (= ${source:Version}),
  ssl-cert (>= 1.0.11),
  ucf,
  ${misc:Depends},
@@ -36,8 +29,12 @@
  systemd (<< 204),
 # pg_buildext moved pg-s-d-all in 204
  postgresql-server-dev-all (<< 204),
+ percona-postgresql-server-dev-all (<< 204),
 Replaces:
  postgresql-server-dev-all (<< 204),
+ percona-postgresql-server-dev-all (<< 204),
+Provides:
+ postgresql-common, percona-postgresql-common
 Description: PostgreSQL database-cluster manager
  The postgresql-common package provides a structure under which
  multiple versions of PostgreSQL may be installed and/or multiple
@@ -54,7 +51,7 @@
  outer joins, multiversion concurrency control, and user-defined types
  and functions.
 
-Package: postgresql-client-common
+Package: percona-postgresql-client-common
 Architecture: all
 Multi-Arch: foreign
 Depends:
@@ -62,7 +59,10 @@
  ${misc:Depends},
  ${pgdg:Depends},
 Recommends:
+ lsb-release,
  ${readline:Recommends},
+#Replaces: percona-postgresql-client-common, postgrsql-client-common
+Provides: postgresql-client-common, percona-postgresql-client-common
 Description: manager for multiple PostgreSQL client versions
  The postgresql-client-common package provides a structure under which
  multiple versions of PostgreSQL client programs may be installed at
@@ -78,13 +78,15 @@
  outer joins, multiversion concurrency control, and user-defined types
  and functions.
 
-Package: postgresql-server-dev-all
+Package: percona-postgresql-server-dev-all
 Architecture: all
 Depends:
  make,
- postgresql-common (>= 117~),
+ percona-postgresql-common (>= 117~),
  ${misc:Depends},
  ${server-dev-all-depends},
+Provides: postgresql-server-dev-all
+Replaces: postgresql-server-dev-all, percona-postgresql-server-dev-all
 Description: extension build tool for multiple PostgreSQL versions
  The postgresql-server-dev-all package provides the pg_buildext script for
  simplifying packaging of a PostgreSQL extension supporting multiple major
@@ -98,13 +100,15 @@
  outer joins, multiversion concurrency control, and user-defined types
  and functions.
 
-Package: postgresql
+Package: percona-postgresql
 Architecture: all
 Depends:
- postgresql-${default-version},
+ percona-postgresql-${default-version},
  ${misc:Depends},
 Suggests:
- postgresql-doc,
+ percona-postgresql-doc,
+Provides: postgresql
+Replaces: postgresql, percona-postgresql
 Description: object-relational SQL database (supported version)
  This metapackage always depends on the currently supported PostgreSQL
  database server version.
@@ -117,12 +121,14 @@
  concurrency control.  Graphical user interfaces and bindings for many
  programming languages are available as well.
 
-Package: postgresql-client
+Package: percona-postgresql-client
 Architecture: all
 Multi-Arch: foreign
 Depends:
- postgresql-client-${default-version},
+ percona-postgresql-client-${default-version},
  ${misc:Depends},
+Provides: postgresql-client
+Replaces: postgresql-client, percona-postgresql-client
 Description: front-end programs for PostgreSQL (supported version)
  This metapackage always depends on the currently supported database
  client package for PostgreSQL.
@@ -135,13 +141,15 @@
  concurrency control.  Graphical user interfaces and bindings for many
  programming languages are available as well.
 
-Package: postgresql-doc
+Package: percona-postgresql-doc
 Architecture: all
 Multi-Arch: foreign
 Section: doc
 Depends:
- postgresql-doc-${default-version},
+ percona-postgresql-doc-${default-version},
  ${misc:Depends},
+Provides: postgresql-doc
+Replaces: postgresql-doc, percona-postgresql-doc
 Description: documentation for the PostgreSQL database management system
  This metapackage always depends on the currently supported PostgreSQL
  database documentation package.
@@ -154,11 +162,13 @@
  concurrency control.  Graphical user interfaces and bindings for many
  programming languages are available as well.
 
-Package: postgresql-contrib
+Package: percona-postgresql-contrib
 Architecture: all
 Depends:
- postgresql-contrib-${default-version},
+ percona-postgresql-contrib-${default-version},
  ${misc:Depends},
+Provides: postgresql-contrib
+Replaces: postgresql-contrib, percona-postgresql-contrib
 Description: additional facilities for PostgreSQL (supported version)
  This metapackage always depends on the currently supported PostgreSQL
  database contrib package.
@@ -171,14 +181,16 @@
  concurrency control.  Graphical user interfaces and bindings for many
  programming languages are available as well.
 
-Package: postgresql-all
+Package: percona-postgresql-all
 Architecture: all
 Depends:
  libecpg-dev,
  libpq-dev,
- postgresql-server-dev-all,
+ percona-postgresql-server-dev-all,
  ${postgresql-all-depends},
  ${misc:Depends},
+Provides: postgresql-all
+Replaces: postgresql-all, percona-postgresql-all
 Description: metapackage depending on all PostgreSQL server packages
  This metapackage depends on all PostgreSQL server packages, in all
  supported versions (excluding documentation and debug symbols). It
