--- postgresql-common.templates	2019-07-04 12:11:15.225670000 +0000
+++ postgresql-common.templates	2019-07-04 12:12:03.601670000 +0000
@@ -1,4 +1,4 @@
-Template: postgresql-common/obsolete-major
+Template: postgresql-common/obsolete-major
 Type: error
 _Description: Obsolete major version ${old}
  The PostgreSQL version ${old} is obsolete, but the server
@@ -18,7 +18,7 @@
  .
  Please see /usr/share/doc/postgresql-common/README.Debian.gz for details.
 
-Template: postgresql-common/ssl
+Template: postgresql-common/ssl
 Type: boolean
 Default: true
 _Description: Enable SSL by default in new PostgreSQL clusters?
@@ -35,7 +35,7 @@
 
 # The last paragraph of this note does not appear in PO files.
 # It contains shell commands and should not be translated.
-Template: postgresql-common/catversion-bump
+Template: postgresql-common/catversion-bump
 Type: note
 #flag:translate:1,2,3
 _Description: PostgreSQL ${version} catalog version changed
