--- supported-versions	2019-12-18 10:55:33.202011549 +0200
+++ supported-versions	2019-12-18 11:25:46.394063621 +0200
@@ -83,10 +83,10 @@
         14.04) /bin/echo -e "9.3" ;;
         16.04|16.10) /bin/echo -e "9.5" ;;
         17.04|17.10) /bin/echo -e "9.6" ;;
-        18.04|18.10) /bin/echo -e "10" ;;
-        19.04|19.10) /bin/echo -e "11" ;;
-        20.04|20.10) /bin/echo -e "12" ;;
-        21.04|21.10) /bin/echo -e "13" ;;
+        18.04|18.10) /bin/echo -e "14" ;;
+        19.04|19.10) /bin/echo -e "14" ;;
+        20.04|20.10) /bin/echo -e "14" ;;
+        21.04|21.10) /bin/echo -e "14" ;;
         22.04|22.10) /bin/echo -e "14" ;;
         *)
             echo "supported-versions: WARNING: Unknown Ubuntu release: $1" >&2
@@ -100,9 +100,9 @@
         6.0*)  /bin/echo -e "8.4" ;; # Squeeze
         7|7.*) /bin/echo -e "9.1" ;; # Wheezy
         8|8.*) /bin/echo -e "9.4" ;; # Jessie
-        9|9.*) /bin/echo -e "9.6" ;; # Stretch
-        10|10.*) /bin/echo -e "11" ;; # Buster
-        11|11.*) /bin/echo -e "13" ;; # Bullseye
+        9|9.*) /bin/echo -e "14" ;; # Stretch
+        10|10.*) /bin/echo -e "14" ;; # Buster
+        11|11.*) /bin/echo -e "14" ;; # Bullseye
         12|12.*) /bin/echo -e "14" ;; # Bookworm
         testing | unstable)
             /bin/echo -e "$DEFAULT" ;;
@@ -118,9 +118,9 @@
         6.0*)  /bin/echo -e "8.4\n9.1" ;; # Squeeze
         7|7.*) /bin/echo -e "9.1\n9.4" ;; # Wheezy
         8|8.*) /bin/echo -e "9.4\n9.6" ;; # Jessie
-        9|9.*) /bin/echo -e "9.6" ;; # Stretch
-        10|10.*) /bin/echo -e "11" ;; # Buster
-        11|11.*) /bin/echo -e "13" ;; # Bullseye
+        9|9.*) /bin/echo -e "14" ;; # Stretch
+        10|10.*) /bin/echo -e "14" ;; # Buster
+        11|11.*) /bin/echo -e "14" ;; # Bullseye
         12|12.*) /bin/echo -e "14" ;; # Bookworm
         testing | unstable)
             /bin/echo -e "$DEFAULT" ;;
