--- control.in	2019-12-20 12:25:46.117381789 +0200
+++ control.in	2019-12-20 16:23:45.149558556 +0200
@@ -1,18 +1,17 @@
-Source: pg-repack
+Source: percona-pg-repack
 Priority: optional
 Section: database
-Maintainer: Debian PostgreSQL Maintainers <team+postgresql@tracker.debian.org>
-Uploaders: Christoph Berg <myon@debian.org>,
-           Adrian Vondendriesch <adrian.vondendriesch@credativ.de>
-Build-Depends: debhelper (>= 9), postgresql-server-dev-all (>= 153~)
+Maintainer: Percona Development Team <info@percona.com>
+Build-Depends: debhelper (>= 9), percona-postgresql-server-dev-all (>= 153~)
 Standards-Version: 4.1.1
 Homepage: https://github.com/reorg/pg_repack
 Vcs-Browser: https://salsa.debian.org/postgresql/pg-repack
 Vcs-Git: https://salsa.debian.org/postgresql/pg-repack.git
 
-Package: postgresql-PGVERSION-repack
+Package: percona-postgresql-PGVERSION-repack
 Architecture: any
-Depends: postgresql-PGVERSION, ${shlibs:Depends}, ${misc:Depends}
+Depends: postgresql-PGVERSION (>= 12.4) | percona-postgresql-PGVERSION (>= 12.4), ${shlibs:Depends}, ${misc:Depends}
+Provides: pg-repack, postgresql-PGVERSION-repack
 Description: reorganize tables in PostgreSQL databases with minimal locks
  pg_repack is a PostgreSQL extension which lets you remove bloat from tables
  and indexes, and optionally restore the physical order of clustered indexes.
