--- Makefile.old        2017-10-14 12:08:28.194278725 +0100
+++ Makefile    2017-10-14 12:08:52.800074297 +0100
@@ -6,7 +6,7 @@
 #  Portions Copyright (c) 2012-2015, The Reorg Development Team
 #
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-16/bin/pg_config
 EXTENSION = pg_repack
 
 .PHONY: dist/$(EXTENSION)-$(EXTVERSION).zip
--- bin/Makefile.old    2017-10-14 12:08:31.034255130 +0100
+++ bin/Makefile        2017-10-14 12:09:01.310003595 +0100
@@ -6,7 +6,7 @@
 #  Portions Copyright (c) 2012-2015, The Reorg Development Team
 #
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-16/bin/pg_config
 
 SRCS = pg_repack.c pgut/pgut.c pgut/pgut-fe.c
 OBJS = $(SRCS:.c=.o)
--- regress/Makefile.old     2017-10-14 12:08:42.563159346 +0100
+++ regress/Makefile    2017-10-14 12:09:19.357853652 +0100
@@ -6,7 +6,7 @@
 #  Portions Copyright (c) 2012-2015, The Reorg Development Team
 #
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-16/bin/pg_config
 
 # version as a number, e.g. 9.1.4 -> 901
 VERSION := $(shell $(PG_CONFIG) --version | sed 's/.* \([[:digit:].]\{1,\}\).*/\1/')
--- lib/Makefile.old    2017-10-14 12:08:39.833182027 +0100
+++ lib/Makefile        2017-10-14 12:09:10.313928790 +0100
@@ -6,7 +6,7 @@
 #  Portions Copyright (c) 2012-2015, The Reorg Development Team
 #
 
-PG_CONFIG ?= pg_config
+PG_CONFIG ?= /usr/pgsql-16/bin/pg_config
 
 # version as a number, e.g. 9.1.4 -> 901
 VERSION := $(shell $(PG_CONFIG) --version | sed 's/.* \([[:digit:].]\{1,\}\).*/\1/')
