--- rules	2019-12-20 12:25:46.117381789 +0200
+++ rules	2019-08-13 11:53:52.000000000 +0300
@@ -8,7 +8,7 @@
 	# nothing to do here, see debian/tests/* instead
 
 override_dh_auto_install:
-	+pg_buildext loop postgresql-%v-repack
+	+pg_buildext loop percona-postgresql-%v-repack
 	# Install an alternative for /usr/bin/pg_repack. We could wrap this
 	# using pg_wrapper, but it doesn't matter which binary gets used (it
 	# figures out the server version by itself), so we just use the one for
@@ -16,8 +16,8 @@
 	set -ex; \
 	for v in $(shell pg_buildext supported-versions); do \
 		i=$$(echo $$v | tr -d .); \
-		echo "update-alternatives --install /usr/bin/pg_repack pg_repack /usr/lib/postgresql/$$v/bin/pg_repack $$i" >> debian/postgresql-$$v-repack.postinst.debhelper; \
-		echo "update-alternatives --remove pg_repack /usr/lib/postgresql/$$v/bin/pg_repack" >> debian/postgresql-$$v-repack.prerm.debhelper; \
+		echo "update-alternatives --install /usr/bin/pg_repack pg_repack /usr/lib/postgresql/$$v/bin/pg_repack $$i" >> debian/percona-postgresql-$$v-repack.postinst.debhelper; \
+		echo "update-alternatives --remove pg_repack /usr/lib/postgresql/$$v/bin/pg_repack" >> debian/percona-postgresql-$$v-repack.prerm.debhelper; \
 	done
 
 override_dh_installdocs:
