diff --git a/src/include/pool.h b/src/include/pool.h
index 56ddbeb1..aec73481 100644
--- a/src/include/pool.h
+++ b/src/include/pool.h
@@ -56,16 +56,16 @@
 #define HBA_CONF_FILE_NAME "pool_hba.conf"
 
 /* pid file directory */
-#define DEFAULT_LOGDIR "/tmp"
+#define DEFAULT_LOGDIR "/var/log/postgresql"
 
 /* Unix domain socket directory */
-#define DEFAULT_SOCKET_DIR "/tmp"
+#define DEFAULT_SOCKET_DIR "/var/run/postgresql"
 
 /* Unix domain socket directory for watchdog IPC */
 #define DEFAULT_WD_IPC_SOCKET_DIR "/tmp"
 
 /* pid file name */
-#define DEFAULT_PID_FILE_NAME "/var/run/pgpool/pgpool.pid"
+#define DEFAULT_PID_FILE_NAME "/var/run/postgresql/pgpool.pid"
 
 /* status file name */
 #define STATUS_FILE_NAME "pgpool_status"
