--- ./pgaudit.c	2019-12-20 23:24:50.000000000 +0200
+++ ./pgaudit.c	2019-12-20 23:24:30.000000000 +0200
@@ -13,7 +13,6 @@
 #include "access/htup_details.h"
 #include "access/sysattr.h"
 #include "access/xact.h"
-#include "access/relation.h"
 #include "catalog/catalog.h"
 #include "catalog/objectaccess.h"
 #include "catalog/pg_class.h"
@@ -38,6 +37,9 @@
 #include "utils/syscache.h"
 #include "utils/timestamp.h"
 #include "utils/varlena.h"
+#if PG_VERSION_NUM >= 120000
+#include "access/relation.h"
+#endif
 
 PG_MODULE_MAGIC;
 
@@ -1007,7 +1009,11 @@
         relOid = rte->relid;
         rel = relation_open(relOid, NoLock);
 
+#if PG_VERSION_NUM >= 120000
         if (!auditLogCatalog && IsCatalogNamespace(RelationGetNamespace(rel)))
+#else
+        if (!auditLogCatalog && IsSystemNamespace(RelationGetNamespace(rel)))
+#endif
         {
             relation_close(rel, NoLock);
             continue;
@@ -1219,7 +1225,11 @@
      * Logging execution of all pg_catalog functions would make the log
      * unusably noisy.
      */
+#if PG_VERSION_NUM >= 120000
     if (IsCatalogNamespace(proc->pronamespace))
+#else
+    if (IsSystemNamespace(proc->pronamespace))
+#endif
     {
         ReleaseSysCache(proctup);
         return;
