--- percona-postgresql-12.spec	2020-05-27 16:21:42.273684493 +0300
+++ percona-postgresql-12.spec	2020-05-27 16:21:18.817769460 +0300
@@ -3,6 +3,7 @@
 %global pgpackageversion 12
 %global prevmajorversion 11
 %global sname postgresql
+%global vname postgresql12
 %global pgbaseinstdir	/usr/pgsql-%{pgmajorversion}
 
 %global beta 0
@@ -81,13 +82,13 @@
 %endif
 
 Summary:	PostgreSQL client programs and libraries
-Name:		%{sname}%{pgmajorversion}
+Name:          percona-postgresql%{pgmajorversion}
 Version:	12.3
-Release:	1PGDG%{?dist}
+Release:	1%{?dist}
 License:	PostgreSQL
 Url:		https://www.postgresql.org/
 
-Source0:	https://download.postgresql.org/pub/source/v%{version}/postgresql-%{version}.tar.bz2
+Source0:       percona-postgresql-12.3.tar.gz
 Source4:	%{sname}-%{pgmajorversion}-Makefile.regress
 Source5:	%{sname}-%{pgmajorversion}-pg_config.h
 %if %{systemd_enabled}
@@ -97,7 +98,7 @@
 %endif
 Source7:	%{sname}-%{pgmajorversion}-ecpg_config.h
 Source9:	%{sname}-%{pgmajorversion}-libs.conf
-Source12:	https://www.postgresql.org/files/documentation/pdf/%{pgpackageversion}/%{sname}-%{pgpackageversion}-A4.pdf
+Source12:	https://www.postgresql.org/files/documentation/pdf/%{pgmajorversion}/%{sname}-%{pgmajorversion}-A4.pdf
 Source14:	%{sname}-%{pgmajorversion}.pam
 Source16:	%{sname}-%{pgmajorversion}-filter-requires-perl-Pg.sh
 Source17:	%{sname}-%{pgmajorversion}-setup
@@ -141,7 +142,7 @@
 %endif
 %if 0%{?rhel} && 0%{?rhel} >= 8
 # Packages come from Appstream:
-BuildRequires:	llvm-devel >= 8.0.1 clang-devel >= 8.0.1
+BuildRequires:	llvm-devel clang-devel
 %endif
 %if 0%{?fedora}
 BuildRequires:	llvm-devel >= 5.0 clang-devel >= 5.0
@@ -264,12 +265,16 @@
 Requires(postun):	initscripts
 %endif
 
-Requires:	%{name}-libs%{?_isa} = %{version}-%{release}
+Requires:	%{name}-libs >= %{version}-%{release}
 
 Requires(post):	%{_sbindir}/update-alternatives
 Requires(postun):	%{_sbindir}/update-alternatives
+Epoch:		1
 
-Provides:	%{sname} >= %{version}-%{release}
+Provides:	%{sname} = %{epoch}:%{version}-%{release}
+Provides:	%{vname} = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname} <= %{version}-%{release}
+Obsoletes:	%{vname} <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
@@ -291,7 +296,6 @@
 
 %package libs
 Summary:	The shared libraries required for any PostgreSQL clients
-Provides:	postgresql-libs = %{pgmajorversion} libpq5 >= 10.0
 %if 0%{?rhel} && 0%{?rhel} <= 6
 Requires:	openssl
 %else
@@ -301,11 +305,17 @@
 Requires:	openssl-libs >= 1.0.2k
 %endif
 %endif
+Provides:	postgresql-libs >= %{version}-%{release}
+Provides:	%{sname}-libs = %{epoch}:%{version}-%{release}
+Provides:	%{vname}-libs = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-libs <= %{version}-%{release}
+Obsoletes:	%{vname}-libs <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
 %endif
+Epoch:		1
 
 %description libs
 The postgresql%{pgmajorversion}-libs package provides the essential shared libraries for any
@@ -315,8 +325,8 @@
 
 %package server
 Summary:	The programs needed to create and run a PostgreSQL server
-Requires:	%{name}%{?_isa} = %{version}-%{release}
-Requires:	%{name}-libs%{?_isa} = %{version}-%{release}
+Requires:	%{name} >= %{version}-%{release}
+Requires:	%{name}-libs >= %{version}-%{release}
 Requires(pre):	/usr/sbin/useradd /usr/sbin/groupadd
 # for /sbin/ldconfig
 Requires(post):		glibc
@@ -337,11 +347,16 @@
 Requires:	/usr/sbin/useradd, /sbin/chkconfig
 %endif
 Provides:	postgresql-server >= %{version}-%{release}
+Provides:	%{vname}-server = %{epoch}:%{version}-%{release}
+Provides:	%{sname}-server = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-server <= %{version}-%{release}
+Obsoletes:	%{vname}-server <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
 %endif
+Epoch:		1
 
 %description server
 PostgreSQL is an advanced Object-Relational database management system (DBMS).
@@ -352,6 +367,11 @@
 %package docs
 Summary:	Extra documentation for PostgreSQL
 Provides:	postgresql-docs >= %{version}-%{release}
+Provides:	%{vname}-docs = %{epoch}:%{version}-%{release}
+Provides:	%{sname}-docs = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-docs <= %{version}-%{release}
+Obsoletes:	%{vname}-docs <= %{version}-%{release}
+Epoch:		1
 
 %description docs
 The postgresql%{pgmajorversion}-docs package includes the SGML source for the documentation
@@ -362,15 +382,19 @@
 
 %package contrib
 Summary:	Contributed source and binaries distributed with PostgreSQL
-Requires:	%{name}%{?_isa} = %{version}-%{release}
-Requires:	%{name}-libs%{?_isa} = %{version}-%{release}
-Requires:	%{name}-server%{?_isa} = %{version}-%{release}
+Requires:	%{name} >= %{version}-%{release}
+Requires:	%{name}-libs >= %{version}-%{release}
 Provides:	postgresql-contrib >= %{version}-%{release}
+Provides:	%{vname}-contrib = %{epoch}:%{version}-%{release}
+Provides:	%{sname}-contrib = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-contrib <= %{version}-%{release}
+Obsoletes:	%{vname}-contrib <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
 %endif
+Epoch:		1
 
 %description contrib
 The postgresql%{pgmajorversion}-contrib package contains various extension modules that are
@@ -378,25 +402,16 @@
 
 %package devel
 Summary:	PostgreSQL development header files and libraries
-Requires:	%{name}%{?_isa} = %{version}-%{release}
-Requires:	%{name}-libs%{?_isa} = %{version}-%{release}
+Requires:	%{name} >= %{version}-%{release}
+Requires:	%{name}-libs >= %{version}-%{release}
 %if %llvm
 %if 0%{?rhel} && 0%{?rhel} == 7
 # Packages come from EPEL and SCL:
-Requires:	llvm5.0-devel >= 5.0 llvm-toolset-7-clang >= 4.0.1
+Requires:	llvm5.0-devel >= 5.0
 %endif
 %if 0%{?rhel} && 0%{?rhel} >= 8
-# Packages come from Appstream:
-Requires:	llvm-devel >= 8.0.1 clang-devel >= 8.0.1
-%endif
-%if 0%{?fedora}
-Requires:	llvm-devel >= 5.0 clang-devel >= 5.0
-%endif
-%if 0%{?suse_version} >= 1315 && 0%{?suse_version} <= 1499
-Requires:	llvm6-devel clang6-devel
-%endif
-%if 0%{?suse_version} >= 1500
-Requires:	llvm5-devel clang5-devel
+# Packages come from EPEL and SCL:
+Requires:	llvm-devel >= 6.0.0
 %endif
 %endif
 %if %icu
@@ -419,11 +434,16 @@
 %endif
 
 Provides:	postgresql-devel >= %{version}-%{release}
+Provides:	%{vname}-devel = %{epoch}:%{version}-%{release}
+Provides:	%{sname}-devel = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-devel <= %{version}-%{release}
+Obsoletes:	%{vname}-devel <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
 %endif
+Epoch:		1
 
 %description devel
 The postgresql%{pgmajorversion}-devel package contains the header files and libraries
@@ -435,7 +455,7 @@
 %if %llvm
 %package llvmjit
 Summary:	Just-in-time compilation support for PostgreSQL
-Requires:	%{name}-server%{?_isa} = %{version}-%{release}
+Requires:	%{name}-server >= %{version}-%{release}
 %if 0%{?rhel} && 0%{?rhel} == 7
 Requires:	llvm5.0 >= 5.0
 %else
@@ -446,11 +466,16 @@
 %endif
 %endif
 Provides:	postgresql-llvmjit >= %{version}-%{release}
+Provides:	%{vname}-llvmjit = %{epoch}:%{version}-%{release}
+Provides:	%{sname}-llvmjit = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-llvmjit <= %{version}-%{release}
+Obsoletes:	%{vname}-llvmjit <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
 %endif
+Epoch:		1
 
 %description llvmjit
 The postgresql%{pgmajorversion}-llvmjit package contains support for
@@ -462,18 +487,23 @@
 %if %plperl
 %package plperl
 Summary:	The Perl procedural language for PostgreSQL
-Requires:	%{name}-server%{?_isa} = %{version}-%{release}
+Requires:	%{name}-server >= %{version}-%{release}
 Requires:	perl(:MODULE_COMPAT_%(eval "`%{__perl} -V:version`"; echo $version))
 %ifarch ppc ppc64
 BuildRequires:	perl-devel
 %endif
-Obsoletes:	postgresql%{pgmajorversion}-pl <= %{version}-%{release}
+Provides:	postgresql%{pgmajorversion}-pl <= %{version}-%{release}
 Provides:	postgresql-plperl >= %{version}-%{release}
+Provides:	%{vname}-plperl = %{epoch}:%{version}-%{release}
+Provides:	%{sname}-plperl = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-plperl <= %{version}-%{release}
+Obsoletes:	%{vname}-plperl <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
 %endif
+Epoch:		1
 
 %description plperl
 The postgresql%{pgmajorversion}-plperl package contains the PL/Perl procedural language,
@@ -485,28 +515,22 @@
 %if %plpython2
 %package plpython
 Summary:	The Python procedural language for PostgreSQL
-Requires:	%{name}%{?_isa} = %{version}-%{release}
-Requires:	%{name}-server%{?_isa} = %{version}-%{release}
+Requires:	%{name} >= %{version}-%{release}
+Requires:	%{name}-server >= %{version}-%{release}
 Obsoletes:	%{name}-pl <= %{version}-%{release}
 Provides:	postgresql-plpython >= %{version}-%{release}
-Provides:	%{name}-plpython2%{?_isa} = %{version}-%{release}
-%if 0%{?rhel} && 0%{?rhel} <= 6
+Provides:	%{name}-plpython2 = %{version}-%{release}
+%if 0%{?rhel} < 7
 Requires:	python-libs
-%endif
-%if 0%{?rhel} == 7 || 0%{?rhel} == 8
-Requires:	python2-libs
-%endif
-%if 0%{?fedora} && 0%{?fedora} <= 31
+%else
 Requires:	python2-libs
 %endif
-%if 0%{?fedora} >= 32
-Requires:	python27
-%endif
-
-%ifarch ppc64 ppc64le
-AutoReq:	0
-Requires:	advance-toolchain-%{atstring}-runtime
-%endif
+Provides:	%{name}-plpython2 = %{version}-%{release}
+Provides:	%{sname}-plpython2 = %{epoch}:%{version}-%{release}
+Provides:	%{vname}-plpython2 = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-plpython2 <= %{version}-%{release}
+Obsoletes:	%{vname}-plpython2 <= %{version}-%{release}
+Epoch:		1
 
 %description plpython
 The postgresql%{pgmajorversion}-plpython package contains the PL/Python procedural language,
@@ -518,8 +542,8 @@
 %if %plpython3
 %package plpython3
 Summary:	The Python3 procedural language for PostgreSQL
-Requires:	%{name}%{?_isa} = %{version}-%{release}
-Requires:	%{name}-server%{?_isa} = %{version}-%{release}
+Requires:	%{name} >= %{version}-%{release}
+Requires:	%{name}-server >= %{version}-%{release}
 Obsoletes:	%{name}-pl <= %{version}-%{release}
 Provides:	postgresql-plpython3 >= %{version}-%{release}
 %if 0%{?rhel} == 7
@@ -528,11 +552,16 @@
 %else
 Requires:	python3-libs
 %endif
+Provides:	%{vname}-plpython3 = %{epoch}:%{version}-%{release}
+Provides:	%{sname}-plpython3 = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-plpython3 <= %{version}-%{release}
+Obsoletes:	%{vname}-plpython3 <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
 %endif
+Epoch:		1
 
 %description plpython3
 The postgresql%{pgmajorversion}-plpython3 package contains the PL/Python3 procedural language,
@@ -544,16 +573,21 @@
 %if %pltcl
 %package pltcl
 Summary:	The Tcl procedural language for PostgreSQL
-Requires:	%{name}%{?_isa} = %{version}-%{release}
-Requires:	%{name}-server%{?_isa} = %{version}-%{release}
+Requires:	%{name} >= %{version}-%{release}
+Requires:	%{name}-server >= %{version}-%{release}
 Requires:	tcl
 Obsoletes:	%{name}-pl <= %{version}-%{release}
 Provides:	postgresql-pltcl >= %{version}-%{release}
+Provides:	%{vname}-pltcl = %{epoch}:%{version}-%{release}
+Provides:	%{sname}-pltcl = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-pltcl <= %{version}-%{release}
+Obsoletes:	%{vname}-pltcl <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
 %endif
+Epoch:		1
 
 %description pltcl
 PostgreSQL is an advanced Object-Relational database management
@@ -564,14 +598,19 @@
 %if %test
 %package test
 Summary:	The test suite distributed with PostgreSQL
-Requires:	%{name}-server%{?_isa} = %{version}-%{release}
-Requires:	%{name}-devel%{?_isa} = %{version}-%{release}
+Requires:	%{name}-server >= %{version}-%{release}
+Requires:	%{name}-devel >= %{version}-%{release}
 Provides:	postgresql-test >= %{version}-%{release}
+Provides:	%{vname}-test = %{epoch}:%{version}-%{release}
+Provides:	%{sname}-test = %{epoch}:%{version}-%{release}
+Obsoletes:	%{sname}-test <= %{version}-%{release}
+Obsoletes:	%{vname}-test <= %{version}-%{release}
 
 %ifarch ppc64 ppc64le
 AutoReq:	0
 Requires:	advance-toolchain-%{atstring}-runtime
 %endif
+Epoch:		1
 
 %description test
 The postgresql%{pgmajorversion}-test package contains files needed for various tests for the
@@ -582,7 +621,7 @@
 %global __perl_requires %{SOURCE16}
 
 %prep
-%setup -q -n %{sname}-%{version}
+%setup -q -n percona-postgresql-%{version}
 %patch1 -p0
 %patch3 -p0
 %patch5 -p0
@@ -893,6 +932,7 @@
 	%{__make} all
 	popd
 %endif
+pushd doc/src; make all; popd
 
 %install
 %{__rm} -rf %{buildroot}
